climakitae/tmy.py

```diff
@@ -1,194 +1,346 @@
     """
     user_options = pn.Card(
             pn.Row(
                 pn.Column(
-                    pn.widgets.StaticText(name="", value='Warming level (°C)'),
-                    pn.widgets.RadioButtonGroup.from_param(warming_levels.param.warmlevel, name=""),
-                    pn.widgets.Select.from_param(warming_levels.param.variable2, name="Data variable"),
+                    pn.widgets.RadioButtonGroup.from_param(tmy_ob.param.tmy_options, name=" "),
+                    pn.widgets.Select.from_param(tmy_ob.param.abs_tmy_options, name="Absolute AMY Options"),
+                    pn.widgets.Select.from_param(tmy_ob.param.diff_tmy_options, name="Difference AMY Options"),
+                    pn.widgets.StaticText(name="", value="Warming level (°C)"),
+                    pn.widgets.RadioButtonGroup.from_param(tmy_ob.param.warmlevel),
+                    pn.widgets.Select.from_param(tmy_ob.param.variable2, name="Data variable"),
                     pn.widgets.StaticText.from_param(selections.param.variable_description),
-                    pn.widgets.Button.from_param(warming_levels.param.reload_data2, button_type="primary", width=150, height=30),
-                    width = 230),
+                    width=230),
                 pn.Column(
-                    pn.widgets.Select.from_param(warming_levels.param.area_subset2, name="Area subset"),
-                    pn.widgets.Select.from_param(warming_levels.param.cached_area2, name="Cached area"),
+                    pn.widgets.Select.from_param(tmy_ob.param.area_subset2, name="Location"),
+                    pn.widgets.Select.from_param(tmy_ob.param.cached_area2, name="Cached area"),
                     location.view,
-                    width = 230)
+                    pn.widgets.Button.from_param(tmy_ob.param.reload_data, button_type="primary", width=150, height=30),
+                    width=230)
                 )
-        , title="Data Options", collapsible=False, width=460, height=500
+        , title=" How do you want to investigate AMY?", collapsible=False, width=460, height=500
+    )
+
+    mthd_bx = pn.Column(
+        pn.widgets.StaticText(
+            value="An average meteorological year is calculated by selecting the 24 hours for every day that best represent multi-model mean conditions during a 30-year period – 1981-2010 for the historical baseline or centered on the year the warming level is reached. Absolute average meteorolgoical year profiles represent data that is not bias corrected, please exercise caution when analyzing. Warming level future aboslute data is based on 30 year period centered around when multi-model mean exceeds a selected warming level (please see explore_warming notebook for further details). The 'severe' AMY is calculated using the 90th percentile of future warming level data at the selected warming level, and is compared to the historical baseline.",
+            width=400
+        ),
```

Comment:
I'd love to see this method box widened a bit to match the width of the panel it's in. Perhaps a width of 500-600 would be more appropriate here.

Reply:
> 

This line is REALLY long lol. Can we do line continuations here?

Reply:
Agree with Eric ^^ just to improve readability of the code

Reply:
Can do x2!

Reply:
Another possibility is to have an `enumerations.py` where you dump large text blobs and then import them into your various python code